spring-boot/proxy/src/main/java/eu/coatrack/proxy/security/ApiKeyAuthTokenVerifier.java

```diff
@@ -20,166 +20,124 @@
  * #L%
  */
 
+import com.netflix.niws.client.http.RestClient;
```

Comment:
We usually use the Spring rest client in CoatRack. Do we need another client?

Reply:
Probably I chose the wrong source for the RestClient when I imported it. It is removed now.